packages/discovery-provider/src/models/users/usdc_purchase.py

```diff
@@ -17,6 +17,11 @@ class PurchaseAccessType(str, enum.Enum):
     download = "download"
 
 
+class PurchaseVendor(str, enum.Enum):
+    user_bank = "user_bank"
+    coinflow = "coinflow"
+
```

Comment:
could add stripe

Reply:
can't!

Reply:
the USDC comes from the user bank not stripe. we can do this soon by adding,.. another memo heheh